libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -192,9 +194,9 @@ export const GridTable = <T extends object>({
 
   // The user / system disables a column
   React.useEffect(() => {
-    if (hiddenColumns) setHiddenColumns(hiddenColumns);
+    setHiddenColumns(hiddenColumns);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [hiddenColumns]);
+  }, [JSON.stringify(hiddenColumns), setHiddenColumns]);
```

Comment:
`React.useEffect(...)` don't like arrays much.  Seem people say to stringify it.  Not sure if it's required.

Reply:
Interesting!